common/djangoapps/entitlements/helpers.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
+
+import pytz
+import sys
+from django.conf import settings
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def is_entitlement_expired(entitlement):
+    """
+    Determines from the policy if an entitlement can be redeemed, if it has not passed the
+    expiration period of policy.expiration_period_days, and has not already been redeemed
+    """
+    utc = pytz.UTC
+    days_since_created = (datetime.utcnow().replace(tzinfo=utc) - entitlement.created).days
```

Comment:
Would using `datetime.now(tz=utc)` eliminate the need for the call to `replace`?

Reply:
Tried it and it works, good call much cleaner to do so that way.